session.go

```diff
@@ -23,8 +24,10 @@ type session struct {
 	net.Conn
 	windowSize    int
 	maxPadding    *big.Int
```

Comment:
What's the benefit of storing it as an int? In the performance critical section, we use it as a big.Int, so might as well store it as such.